kolibri/core/assets/src/utils/cookieUtils.js

```diff
@@ -0,0 +1,13 @@
+export function setCookie(key, value, expiresInMs) {
+  const currentDate = new Date();
+  currentDate.setTime(currentDate.getTime() + expiresInMs);
+  const expires = `expires=${currentDate.toUTCString()}`;
+  window.document.cookie = `${key}=${value}; ${expires}; path=/`;
+}
+
+export function getCookie(key) {
+  return window.document.cookie
+    .split('; ')
+    .find(cookie => cookie.startsWith(`${key}=`))
+    ?.split('=')[1];
+}
```

Comment:
I realize these utilities were added in a separate PR, but since they're becoming global utilities, we should think about using a library for this to cover edge cases. For instance, the cookie value needs to be URI encoded in case it has commas, semicolons or whitespace. There may be other edge cases I'm not recalling

Reply:
Sounds like we've used `cookiejs` ([link](https://www.npmjs.com/package/cookiejs)) before, but removed it when we no longer needed it. Looking at its source, it seems to have a few tricks that likely cover edge cases like the one I mentioned.

Reply:
Thanks @bjester  for raising this. I will proceed to use the `cookiejs`. I was initially hesitant to introduce a new library into the project as i felt it would be an overkill to a really basic functionality. But since it covers for edge cases better, then we can introduce it back

Reply:
@bjester [`js-cookie`](https://www.npmjs.com/package/js-cookie) seems more robust and documentation more clearer. Any thoughts?

Reply:
https://github.com/learningequality/kolibri/pull/10164 is tracking the library addition changes